cli/start_file_writer.py

```diff
@@ -0,0 +1,222 @@
+import argparse
+import os
+import signal
+import sys
+import time
+from datetime import datetime, timedelta
+from time import time as current_time
+from typing import Optional, Tuple
+
+from file_writer_control import JobHandler, JobState, WorkerCommandChannel, WriteJob
+
+JOB_HANDLER: JobHandler
+ACK_TIMEOUT: float
+
+
+def cli_parser() -> argparse.Namespace:
+    """
+    Parser for the command line interface.
+    """
+
+    fw_parser = argparse.ArgumentParser(
+        fromfile_prefix_chars="@", description="FileWriter Starter"
+    )
+
+    fw_parser.add_argument(
+        "-f",
+        "--filename",
+        metavar="filename",
+        type=str,
+        required=True,
+        help="Name of the output file, e.g., `<filename>.nxs`.",
+    )
+    fw_parser.add_argument(
+        "-c",
+        "--config",
+        metavar="json_config",
+        type=str,
+        required=True,
+        help="Path to JSON config file.",
+    )
+    fw_parser.add_argument(
+        "-b",
+        "--broker",
+        metavar="kafka_broker",
```

Comment:
should we have a default here instead: if missing, default to localhost:9092 ?

Reply:
Oh, great catch! yes, thanks.